support-frontend/assets/pages/supporter-plus-landing/components/__snapshots__/paymentTsAndCs.test.tsx.snap

```diff
@@ -10,11 +10,11 @@ exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product GuardianW
 
 exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product GuardianWeeklyRestOfWorld for region Annual correctly 1`] = `" By proceeding, you are agreeing to our Terms & Conditions. To find out what personal data we collect and how we use it, please visit our Privacy Policy.All card payments are powered by Stripe. Read the Stripe Privacy Policy and Terms and conditions."`;
 
-exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product HomeDelivery for region Monthly correctly 1`] = `"Your first payment will be taken on Sunday, September 21, 2025 when your first newspaper is delivered. We will share your contact and subscription details with our fulfilment partners.By proceeding, you are agreeing to our Terms & Conditions. To find out what personal data we collect and how we use it, please visit our Privacy Policy.All card payments are powered by Stripe. Read the Stripe Privacy Policy and Terms and conditions."`;
+exports[`Payment Ts&Cs Snapshot comparison paymentTs&Cs render product HomeDelivery for region Monthly correctly 1`] = `"Your first payment will be taken on Sunday, September 21, 2025 when your first newspaper is delivered. You can cancel your subscription at any time before your next renewal date. Cancellation will take effect at the end of your current payment period. To cancel, use the contact details listed on our Help Centre. We will share your contact and subscription details with our fulfilment partners.By proceeding, you are agreeing to our Terms & Conditions. To find out what personal data we collect and how we use it, please visit our Privacy Policy.All card payments are powered by Stripe. Read the Stripe Privacy Policy and Terms and conditions."`;
```

Comment:
What's the difference between the two versions of the copy here? Is the added copy what we show for paper+ subs and it's showing up here because the snapshots didn't cover paper+ before?

Reply:
Yes thats exactly it, the original copy is non-plus, the new is plus.

Reply:
Yes, the original snapshots covered HomeDelivery/SubscriptionCard/NationalDelivery with (erronous) `Monthly` ActiveRatePlanKey giving th old copy. The new snapshots with the `Monthly` ActiveRatePlanKey display the correct/new copy as shown in the existing checkout below -> 
<img width="843" height="222" alt="image" src="https://github.com/user-attachments/assets/eaa768f9-9d57-4cfb-b21b-da96538e8084" />